src/run_webhook_api.py

```diff
@@ -38,9 +38,26 @@ async def telegram(request: Request) -> Response:
     return Response()
 
 
+async def trello_call_back(request: Request) -> Response:
+    """
+    Plug func catching trello post request
+    :param request: Trello request
+    :return: Response "ok"
+    """
+    if request.method == "HEAD":
+        return Response()
+    elif request.method == "POST":
+        # if request.headers and await request.json():
+        #     message = "Информация с сайта Trello успешно получена и обработана."
+        #     await request.app.state.bot_app.bot.send_message(chat_id=config.CHAT_ID, text=message)
+        return Response("Message received")
+    return Response("Something wrong", status_code=405)
+
+
 routes = [
     Route("/telegram", telegram, methods=["POST"]),
     Route("/health", health, methods=["GET"]),
+    Route("/trelloCallback", trello_call_back, methods=["POST", "HEAD"]),
```

Comment:
Давай немного переименуем методы и добавим это в документик с соглашением
```suggestion
    Route("/telegramWebhookApi", telegram_webhook_api, methods=["POST"]),
    Route("/healthcheck", healthcheck_api, methods=["GET"]),
    Route("/trelloWebhookApi", trello_webhook_api, methods=["POST", "HEAD"]),
```
что для всех апи нужно добавлять окончание _апи